cmd/migrate/migrate.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/insolar/insconfig"
+	"gopkg.in/gormigrate.v1"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+
+	"github.com/insolar/block-explorer/configuration"
+	"github.com/insolar/block-explorer/etl/models"
+	"github.com/insolar/block-explorer/instrumentation/belogger"
+)
+
+func main() {
+	cfg := &configuration.DB{}
+	params := insconfig.Params{
+		EnvPrefix:        "migrate",
+		ConfigPathGetter: &insconfig.DefaultPathGetter{},
+	}
+	insConfigurator := insconfig.New(params)
+	if err := insConfigurator.Load(cfg); err != nil {
+		panic(err)
+	}
+	fmt.Println("Starts with configuration:\n", insConfigurator.ToYaml(cfg))
+
+	ctx := context.Background()
+	log := belogger.FromContext(ctx)
+
+	db, err := gorm.Open("postgres", cfg.URL)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+
+	db.LogMode(true)
+	db.SetLogger(belogger.NewGORMLogAdapter(log))
+
+	m := gormigrate.New(db, gormigrate.DefaultOptions, migrations())
+
+	if err = m.Migrate(); err != nil {
+		log.Fatalf("Could not migrate: %v", err)
+	}
+	log.Info("migrated successfully!")
+}
+
+func migrations() []*gormigrate.Migration {
+	return []*gormigrate.Migration{
+		{
+			ID: "202005180423",
+			Migrate: func(tx *gorm.DB) error {
+				type Pulse struct {
```

Comment:
Definitely wrong usage of gorm. This tags must be used on exported structures. I can say, this tags must be used in types.go

Reply:
It's a good practice to copy the struct inside migration, so side effects are prevented if the original struct changes

Reply:
And tags needed here to create PK for tables